packages/orchestration/src/exos/local-chain-account-kit.js

```diff
@@ -79,8 +79,7 @@ export const prepareLocalChainAccountKit = (
   /**
    * Make an object wrapping an LCA with Zoe interfaces.
    */
-  const makeLocalChainAccountKit = prepareExoClassKit(
-    baggage,
+  const makeLocalChainAccountKit = zone.exoClassKit(
     'LCA Kit',
```

Comment:
I wonder how to be sure we have no kind tag collisions.

Reply:
integration tests. kind tags have to be unique per baggage, not per system